NEWS.md

```diff
@@ -52,6 +52,9 @@
 
 ## Minor improvements and bug fixes
 
+* `person` now supports all [MARC Relator](http://www.loc.gov/marc/relators/relaterm.html "MARC Relator roles")
```

Comment:
I think you can remove ` "MARC Relator roles"`

Reply:
I've now removed it, in commit c771b0f.